[Bulk] Broken Link fixes (#878)
MicrosoftDocs/biztalk-docs: biztalk/technical-guides/post-configuration-database-optimizations1.md

- Database auto-growth should be set to a fixed number of megabytes instead of to a percentage (specify file growth **In Megabytes**). This should be done so if auto-growth occurs, it does so in a measured fashion which reduces the likelihood of excessive database growth. The growth increment should generally be no larger than 100 MB (for large files), 10 MB (for medium-sized files), or 1 MB (for small files).

> - When SQL Server increases the size of a file, the new space must first be initialized before it can be used. This is a blocking operation that involves filling the new space with empty pages. SQL Server 2005 running on Windows Server 2003 or later supports “instant file initialization.” This can greatly reduce the performance impact of a file growth operation. For more information, see [Database File Initialization](https://go.microsoft.com/fwlink/?LinkId=132063) in the SQL Server 2008 documentation. This topic provides steps for enabling instant file initialization.

Reviewer: ```suggestion
- When SQL Server increases the size of a file, the new space must first be initialized before it can be used. This is a blocking operation that involves filling the new space with empty pages. SQL Server 2005 running on Windows Server 2003 or later supports “instant file initialization.” This can greatly reduce the performance impact of a file growth operation. For more information, see [SQL Server 2008 - Database File Initialization](/sql/relational-databases/databases/database-instant-file-initialization?redirectedfrom=MSDN&view=sql-server-ver16). This topic provides steps for enabling instant file initialization.
```

## Configure Purging and Archiving of Tracking Data
 Follow these steps to ensure that purging and archiving of tracking data is configured correctly:

> -   Ensure the SQL Agent job “DTA Purge and Archive” is properly configured, enabled, and successfully completing. For more information, see "How to Configure the DTA Purge and Archive Job" in the BizTalk Server documentation.

Reviewer: ```suggestion
-   Ensure the SQL Agent job “DTA Purge and Archive” is properly configured, enabled, and successfully completing. For more information, see [Configure the DTA Purge and Archive Job](/biztalk/core/how-to-configure-the-dta-purge-and-archive-job).
```

> -   Ensure the job is able to purge the tracking data as fast as the incoming tracking data is generated. For more information, see "Measuring Maximum Sustainable Tracking Throughput" in the BizTalk Server 2006 R2 documentation.

Reviewer: ```suggestion
-  Make sure that the job can purge the tracking data as fast as the incoming tracking data is generated. For more information, see [Measuring Maximum Sustainable Tracking Throughput](/biztalk/core/measuring-maximum-sustainable-tracking-throughput).
```

> -   Review the soft purge and hard purge parameters to ensure you are keeping data for the optimal length of time. For more information, see "Archiving and Purging the BizTalk Tracking Database" in the BizTalk Server documentation.

Reviewer: ```suggestion
-  Review the soft purge and hard purge parameters to ensure you are keeping data for the optimal length of time. For more information, see [Archive and Purge the BizTalk Tracking Database](/biztalk/core/archiving-and-purging-the-biztalk-tracking-database).
```

> -   If you only need to purge the old data and do not need to archive it first, change the SQL Agent job to call the stored procedure “dtasp_PurgeTrackingDatabase.” For more information, see "How to Purge Data from the BizTalk Tracking Database" in the BizTalk Server documentation.

Reviewer: ```suggestion
-  If you only need to purge the old data and don't need to archive first, change the SQL Agent job to call the stored procedure named **dtasp_PurgeTrackingDatabase**. For more information, see [Purge Data from the BizTalk Tracking Database](/biztalk/core/how-to-purge-data-from-the-biztalk-tracking-database).
```
